lib/config/guiconfiguration.go

```diff
@@ -12,15 +12,28 @@ import (
 	"strings"
 )
 
+const (
+	AuthModeSimple = "simple"
+	AuthModeLDAP   = "ldap"
+)
```

Comment:
These consts probably should be their own types and enums, like FolderType et al.